SPECIFICATION.md

```diff
@@ -77,65 +78,62 @@ can be deserialized back into the same UTF-8 string.
 
 ## Evaluation
 
-Evaluation of access expressions performs a combination of [set][2] existence
-checks and [boolean algebra][3]. Access expression use the following from
-boolean algebra.
+The evaluation process combines [set][2] existence checks with [boolean algebra][3]. Specifically, 
+AccessExpressions use:
 
- * The symbol `&` in an access expression represents [logical conjunction][4]
-   which is represented in a boolean algebra as `∧`.
- * The symbol `|` in an access expression represents [logical disjunction][5]
-   which is represented in a boolean algebra as `∨`.
+ * The symbol `&` for [logical conjunction][4] (`∧` in boolean algebra).
+ * The symbol `|` for [logical disjunction][5] (`∨` in boolean algebra).
 
-When evaluating an access expression set existence checks are done against an
-entities set of authorizations. The following is an algorithm for evaluation an
-access expression.
+When evaluating an AccessExpression set, existence checks are done against an
```

Comment:
In the following `set` went with `existence`, so should not put a comma between.  Maybe the word `set` could just be dropped.


```suggestion
When evaluating an AccessExpression, existence checks are done against an
```